elements/gnome-build-meta.bst

```diff
@@ -0,0 +1,23 @@
+# This is derived from gnome-build-meta 48.2, ensure to update it when rebasing.
+
+kind: junction
+
+sources:
+- kind: git_repo
+  url: gnome:gnome-build-meta.git
+  track: 48*
+  ref: 48.2-0-g5d5d8f32365d766001d9b48a6643d6962a9dc94a
+
+config:
+  overrides:
+    freedesktop-sdk.bst: freedesktop-sdk.bst
```

Comment:
Now, I understand how to patch the components in freedesktop-sdk with this junction's overrides.  Thank you!